src/test/jtx/Env_test.cpp

```diff
@@ -250,7 +250,7 @@ class Env_test : public beast::unit_test::suite
     {
         using namespace jtx;
 
-        Env env(*this);
+        Env env{*this, supported_amendments() | fixDisabledRegularKey};
```

Comment:
Because `fixDisabledRegularKey` is in `supported_amendments()`, is it necessary to explicitly add it in here?

Reply:
I want to be robust against changes in `supported_amendments`. To be honest, I'm surprised that the tests aren't already like this, because it meant I had to add `- fixDisabledRegularKey` to some existing tests. I prefer that new code not have to change existing tests.

Reply:
Amendments aren't removed from `supported_amendments` very often, so I hadn't really considered the need. I agree about new code not changing existing tests in general, but when you're changing functionality, sometimes you gotta do what you gotta do. :smile: